test/libsolidity/ASTJSON/pragma_experimental_solidity.sol

```diff
@@ -1,4 +0,0 @@
-pragma experimental solidity;
-// ====
-// EVMVersion: >=constantinople
-// ----
```

Comment:
Can we keep this test with
```
// ----
// failAfter: Parsed
```

Reply:
Yes. I didn't think of that oprtion :smile: